src/Sylius/Behat/Page/Admin/Product/CreateSimpleProductPage.php

```diff
@@ -223,11 +243,17 @@ protected function getDefinedElements(): array
             'shipping_required' => '#sylius_product_variant_shippingRequired',
             'slug' => '#sylius_product_translations_%locale%_slug',
             'tab' => '.menu [data-tab="%name%"]',
+            'taxonomy' => 'a[data-tab="taxonomy"]',
             'toggle_slug_modification_button' => '.toggle-product-slug-modification',
         ]);
     }
 
-    private function selectElementFromAttributesDropdown(string $id): void
+    private function openTaxonBookmarks(): void
+    {
+        $this->getElement('taxonomy')->click();
+    }
+
+    private function selectElementFromAttributesDropdown(int $id): void
```

Comment:
For some reason, changing `$id` type from `string` to `int` breaks the step :( I would leave it as it was and try to debug it later